readme/_i18n/fr_FR/desktop.md

```diff
@@ -1,5 +1,15 @@
 # Application de bureau
 
-<img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/DemoDesktop.png" style="width: 100%">
+<img src="/images/DemoDesktop.png" style="width: 100%">
+
+The above screenshot shows a note open in Joplin's Markdown editor. The numbers label different parts of Joplin's UI:
+
+1. Notebooks
+2. Tags
+3. Sidebar
+4. Note list
+5. Note editor in Markdown
+6. Note viewer (shows the rendered Markdown)
+7. Toggle editors — switches between Markdown and Rich Text.
```

Comment:
Rather than leaving `fr_FR` with just the screenshot without explanation, I've included the English text.

Reply:
I forgot exactly how this works, and I'm not even sure we still use this. For now, please ignore this file